anime_downloader/sites/darkanime.py

```diff
@@ -0,0 +1,44 @@
+import json, re
+
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+from anime_downloader.sites import helpers
+
+class DarkAnime(Anime, sitename = 'darkanime'):
+
+    sitename = 'darkanime'
+
+    @classmethod
+    def search(cls, query):
+
+        soup = helpers.soupify(helpers.get('https://app.darkanime.stream/api/v1/animes', params={'term': query}).json()['animesHtml'])
+        soup = soup.find_all('a', href=True)
+        results = [[x.find('h3').text.strip(), 'https://app.darkanime.stream' + x['href']] for x in soup]

```

Comment:
Why do you define results and loop through that instead of doing searchresult() directly on the info?

Reply:
because i like that